validation/chOpenApiValidator.go

```diff
@@ -27,7 +27,7 @@ var (
 
 // CHValidator provides an interface to interact with the CH Validator.
 type CHValidator interface {
-	ValidateRequestAgainstOpenApiSpec(httpReq *http.Request, openApiSpec string) ([]byte, error)
+	ValidateRequestAgainstOpenApiSpec(httpReq *http.Request, openApiSpec, contextId string) ([]byte, error)
```

Comment:
Would it be easier to pass the config object here instead of openApiSpec and contextId ?